translator/trace/zipkin/protospan_to_zipkinv1.go

```diff
@@ -210,79 +212,74 @@ func serviceNameOrDefault(node *commonpb.Node, defaultServiceName string) string
 	return node.ServiceInfo.Name
 }
 
-func OCSpanDataToZipkin(
+func OCSpanProtoToZipkin(
 	node *commonpb.Node,
-	s *trace.SpanData,
+	resource *resourcepb.Resource,
+	s *tracepb.Span,
 	defaultServiceName string,
-) (zc zipkinmodel.SpanModel) {
+) (*zipkinmodel.SpanModel, error) {
+	if s == nil {
+		return nil, errNilSpan
+	}
+
+	var attrMap map[string]*tracepb.AttributeValue
+	if s.Attributes != nil {
+		attrMap = s.Attributes.AttributeMap
+	}
 
 	// Per call to zipkinEndpointFromAttributes at most 3 attribute keys can be
 	// made redundant, give a hint when calling make that we expect at most 6
 	// items on the map.
```

Comment:
I don't think that comment is accurate anymore. I'd probably delete it (and change the map capacity below to 9)

Reply:
I think it is 8. Changed.